package-lock.json

```
@@ -2549,4 +2549,4 @@
       "integrity": "sha512-r3vXyErRCYJ7wg28yvBY5VSoAF8ZvlcW9/BwUzEtUsjvX/DKs24dIkuwjtuprwJJHsbyUbLApepYTR1BN4uHrg=="
     }
   }
-}
+}
```

This file should be removed :)

---

Do you mean i should delete the package-lock.json file completely Sir?

---

Yes, you can remove it completely! Also, you don't have to call us Sir or Ma'am, you can just refer to us by our github handles 😄

---

Haha Okay no problem

---

@Lontchi12 Sorry I wasn't clear before! I meant you can remove the `package-lock.json` from the diff, not that you should delete the `package-lock.json` file. There should not be any changes to the `package-lock.json` file. Let me know if this makes sense, the rest looks good :)

---

Okay @gs0510 i hope i have done that now

---

@Lontchi12 This file is still here! 😁 It shouldn't be but because there is a change in line ending (I suspect you are using Windows) it still shows up when you do a `git add .` even though you haven't changed anything. If you search for how to remove a modified file from pull request, it should guide you adequately on how to remove this file so it doesn't show up on the diff here.

---

Hello @gs0510 i've tried to revert the file to the original one but there is no change, I have tried many times to update the package-lock.json file to the one on the upstream main branch but it's still showing that nothing has been changed.
Please look at the line 2552 and you'll see that it's the same with the one of my pull request.

---

I believe it's missing a newline @Lontchi12 -- Github does the opposite of what you think. It shows the 🚫-looking symbol when there's no newline otherwise it is implicit. So your file has no blank newline at the end where as the original did (I think). So probably something like the following is needed:

```suggestion
}

```